### ISSUE TITLE
false postive with defer

### ISSUE THREAD

**Issue by ldez** [ykadowak/zerologlint#3]
```go
package main

import "github.com/rs/zerolog/log"

func main() {
	defer log.Info().Msg("Server stopped")
}
```

```console
$ golangci-lint run -Ezerologlint
main.go:6:16: zerologlinter: must be dispatched by Msg or Send method (zerologlint)
        defer log.Info().Msg("Server stopped")
                      ^
```

**Comment by ykadowak** [comment]
Thanks for the report. Will release and make PR for `golangci-lint` tomorrow.

**Comment by ldez** [comment]
You don't need to open a PR: you just have to create a tag and our bot will manage the update automatically.

**Comment by ykadowak** [comment]
Oh great! Just released `v0.1.2`.
